roles/edpm_nova/molecule/vagrant/molecule.yml

```diff
@@ -0,0 +1,37 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: vagrant
+  provider:
+    name: libvirt
+  provision: no
+  parallel: true
+  default_box: 'generic/rocky9'
```

Comment:
the usage is `rocky` is intended here?

Reply:
ish i try to avoid usign centos9s normally as its less stable then rocky9 but the rocky9 libvirt repo was offline last week so i meant to change it to centos9s as i did for libvirt

what ill do is update this back to centos9s and leave rocky commented and then people can use choose locally.


defaulting to centos9s would be closer to what we use for the rest of development so that probably a better default.